### PR TITLE
Update public_share.md (zenodo loader) 

### PR DESCRIPTION
**Pull request by LukasLohoff**
API documentation on the zenodo loader to `public_share.md`

- distinction sciebo/zenodo
- required parameters
- examples
- errors

### CODE REVIEW

File: docs/compendium/public_share.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Public share
 
-Upload an unvalidated research compendium by submitting a link to a cloud resource. Currently, only sciebo (https://www.sciebo.de/en/) is supported.
+Upload an unvalidated research compendium by submitting a link to a cloud resource. Currently, sciebo (https://www.sciebo.de/en/) and Zenodo (https://sandbox.zenodo.org) is supported.
```
Comment (nuest) [inline docs/compendium/public_share.md]: We support both sandbox.zenodo.org and zenodo.org, right? As for the case "which one to use if only the zenodo id is given", that can be detailed below. Here, please mention both.

File: docs/compendium/public_share.md [hunk 2]
```diff
@@ -52,10 +70,93 @@ Depending on the file structure, the public share contents are treated different
...
+2. DOI
+
+```bash
+curl -d "content_type=compendium_v1" \
+    -d "doi=10.5072/zenodo.69114"  \
+    --cookie "connect.sid=<code string here>" \
+     http://…:8088/api/v2/compendium
+```
+
+3. Zenodo Record ID
```
Comment (nuest) [inline docs/compendium/public_share.md]: Please add a note here that if only the record ID is provided, then a default of Zenodo and Zenodo sandbox is used as selected by the API maintainer.

**Comment by nuest** [comment]
@LukasLohoff good job, just two minor comments.

@7048730 fyi, this is how the API for loading from Zenodo currently looks.

**Will be merged when deployed online.**

File: docs/compendium/public_share.md (hunk 2 above)
Comment (LukasLohoff) [inline docs/compendium/public_share.md]: Alright, it's fixed

File: docs/compendium/public_share.md (hunk 1 above)
Comment (LukasLohoff) [inline docs/compendium/public_share.md]: Ok, I replaced it with a list of supported repositories

File: docs/compendium/public_share.md [hunk 3]
```diff
@@ -15,7 +15,7 @@ Upon successful download from the public share, the `id` for the new compendium
 curl -d "content_type=compendium_v1" \
     -d "share_url=https://uni-muenster.sciebo.de/index.php/s/G8vxQ1h50V4HpuA"  \
     --cookie "connect.sid=<code string here>" \
-     http://…:8088/api/v2/compendium
+     http://…:8088/api/v1/compendium
```
Comment (nuest) [inline docs/compendium/public_share.md]: Actually, the port should be left our here, as the API spec is for the whole API, not for one microservice. Please update :-) (and other URLs, too).

**Comment by nuest** [comment]
This represents the code of the current Docker images so we'll merge this, but this is _not_ yet on the public test server.

**Comment by nuest** [comment]
Thanks @LukasLohoff !

**Pull request merged**